app/_hub/kong-inc/prometheus/index.md

```diff
@@ -98,6 +98,10 @@ Access-Control-Allow-Origin: *
 # TYPE kong_bandwidth counter
 kong_bandwidth{type="egress",service="google",route="google.route-1"} 1277
 kong_bandwidth{type="ingress",service="google",route="google.route-1"} 254
+# HELP kong_nginx_timers Number of nginx timers
+# TYPE kong_nginx_timers gauge
+kong_nginx_timers{state="running"} 3
+kong_nginx_timers{state="pending"} 1
```

Comment:
@limkevinkuan could you please confirm that these lines are correct?

Reply:
Yep, they're good.